README.md

```diff
@@ -46,6 +46,18 @@ other tools for measuring time (which are non-deterministic and produce reports
 - Windows (WSL): `apt install time timeout`
 - macOS: `brew install gnu-time coreutils`
 
+If you would like to have autocomplete for `sinol-make` commands,
+add the following line to your `~/.bashrc` (or `~/.zshrc`) file:
+
+```shell
+eval "$(register-python-argcomplete sinol-make)"
+```
+or run the following command:
```

Comment:
```suggestion
run the following command and refresh the shell (e.g. by opening a new terminal):
```

Reply:
Even though the command requires refreshing the shell, it's easier to do and requires less technical knowledge.